From c53c2695b13ff6bfe8745aa3855f2c53fd4056ed Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Sat, 24 Feb 2018 15:36:06 +0800
Subject: [PATCH] Revert "arm64: rockchip_defconfig: remove
 CONFIG_PM_AUTOSLEEP"

This reverts commit 3c4e6c90408509bbfc52a4fd0eb34b6e784ed100.

Failed to pass vintf compatibility test.

Change-Id: I2455a12a1174b8975cac53d0423ac3a7983f1379
---
 arch/arm64/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 0887a4ecfa73..01541e6f1386 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -75,6 +75,7 @@ CONFIG_ARM64_SW_TTBR0_PAN=y
 # CONFIG_EFI is not set
 # CONFIG_COREDUMP is not set
 CONFIG_COMPAT=y
+CONFIG_PM_AUTOSLEEP=y
 CONFIG_PM_WAKELOCKS=y
 CONFIG_PM_WAKELOCKS_LIMIT=0
 # CONFIG_PM_WAKELOCKS_GC is not set
-- 
2.35.3

